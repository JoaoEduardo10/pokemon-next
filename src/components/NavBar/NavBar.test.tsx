import { render } from '@testing-library/react';
import { NavBar } from '.';

describe('<NavBar />', () => {
	it('should render a NaveBar', () => {
		const { container } = render(<NavBar />);

		expect(container).toMatchSnapshot();
	});
});