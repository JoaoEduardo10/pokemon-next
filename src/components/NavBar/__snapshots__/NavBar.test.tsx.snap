// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<NavBar /> should render a NaveBar 1`] = `
<div>
  <nav
    class="navBar"
  >
    <div
      class="logo"
    >
      <img
        alt="pokeball"
        data-nimg="1"
        decoding="async"
        height="30"
        loading="lazy"
        src="/_next/image?url=%2Fimages%2Fpokeball.png&w=64&q=75"
        srcset="/_next/image?url=%2Fimages%2Fpokeball.png&w=32&q=75 1x, /_next/image?url=%2Fimages%2Fpokeball.png&w=64&q=75 2x"
        style="color: transparent;"
        width="30"
      />
      <h1>
        PokeNext
      </h1>
    </div>
    <ul
      class="link_itens"
    >
      <li>
        <a
          href="/"
        >
          Home
        </a>
      </li>
      <li>
        <a
          href="/about"
        >
          Sobre
        </a>
      </li>
    </ul>
  </nav>
</div>
`;
